src/amqp.c

```diff
@@ -282,6 +298,17 @@ amqp_bytes_t amqp_declare_new_queue(const char *name)
 	return queue;
 }
 
+/**
+ * amqp_set_queue_to_consume:
+ * @queue: queue declared
+ * @exchange: exchange to be declared
+ * @routing_key: routing key to bind
+ *
+ * This function declare a exchange and bind a routing key to a queue to be a
```

Comment:
Remove "This. Can be "Declare..." or "Function to declare..."

Reply:
Done